### ISSUE TITLE
Change Semester shown in Termine and ext/Termine

### ISSUE THREAD

**Issue by Beckicious** [i4Ds/ProStudCreator#297]
Mail Sibylle:
```
m. E. Nur genau bis alle Events abgeschlossen sind. Neu gibt es beim Wechsel im Herbst keine 
Überschneidungen mehr, Im Frühling allerdings schon, daher der Vorschlag dass man die
Wechsel auf die Abgabe der Bachelorarbeit, resp. Nach der Verteidigung taktet (plus ca. 2 Wochen). 

Entweder im Mai oder Ende Frühlingssemester und Beginn Herbstsemester wäre mein Vorschlag. 
```

**Comment by dgruntz** [comment]
Mail Dominik:

````
Ich unterstütze diesen Vorschlag. Ich denke der Wechsel auf Anfang Semester war sinnvoll als es 
noch die IP5 Langprojekte gab, denn diese wurden Ende Dezember abgegeben, und da macht ein
Wechsel auf Anfang Semester Sinn.

Sinnvoll ist ein Wechsel sobald alle Termine einer Durchführung vorbei sind.
Der letzte Termin ist aktuell das Vereidigungsfenster. Der Vorschlag von Sibylle, Ende 
Verteidigung + 2 Wochen finde ich gut.

Falls das ungünstig für die Programmierung ist, dann könnte man auch den Wechsel auf
Abgabe + 7 Wochen setzen (das ist in etwa derselbe Termin, denn nach der Abgabe haben
wir 2-3 Wochen Korrektur dann 2 Wochen Verteidigung).
````

**Comment by Beckicious** [comment]
implemented in 8738bd55b6f697fbc81cb733a06723c5ada00c45
